tests/Command/GenerateTemplatesCommand/expected_output/families/batteries.json

```
@@ -0,0 +1,130 @@
+{
+  "code": "batteries",
+  "labels": { "en_US": "Batteries" },
+  "description": {
+    "en_US": "Power up your devices with our 'Batteries' DPP template - effortlessly manage your battery inventory, ensuring uninterrupted performance and endless productivity whenever you need it most."
+  },
+  "attribute_as_main_media": "main_image",
+  "attribute_as_label": "product_title",
+  "attributes": [
+    {
+      "code": "EAN",
+      "labels": { "en_US": "Main Identifier" },
+      "type": "pim_catalog_identifier",
+      "scopable": false,
+      "localizable": false,
+      "unique": true
+    },
+    {
+      "code": "product_title",
+      "labels": { "en_US": "Product title" },
+      "type": "pim_catalog_text",
+      "scopable": false,
+      "localizable": true,
+      "unique": false
+    },
+    {
+      "code": "main_image",
+      "labels": { "en_US": "Main image" },
+      "type": "pim_catalog_image",
+      "scopable": false,
+      "localizable": false,
+      "unique": false
+    },
+    {
+      "code": "eu_declaration_of_conformity",
+      "labels": { "en_US": "EU declaration of conformity" },
+      "type": "pim_catalog_text",
+      "scopable": false,
+      "localizable": true,
+      "unique": false,
+      "validation_rule": "url"
```

IMO we should have `validation_rule` key inside every `pim_catalog_text` attribute even if it is not referenced in the XLSX file. That's what we do with `pim_catalog_number` with decimals and negative allowed by setting `false` as default value.

```suggestion
      "code": "product_title",
      "labels": { "en_US": "Product title" },
      "type": "pim_catalog_text",
      "scopable": false,
      "localizable": true,
      "unique": false,
      "validation_rule": null
    },
    {
      "code": "main_image",
      "labels": { "en_US": "Main image" },
      "type": "pim_catalog_image",
      "scopable": false,
      "localizable": false,
      "unique": false
    },
    {
      "code": "eu_declaration_of_conformity",
      "labels": { "en_US": "EU declaration of conformity" },
      "type": "pim_catalog_text",
      "scopable": false,
      "localizable": true,
      "unique": false,
      "validation_rule": "url"
```